seatunnel-connectors-v2/connector-iceberg/src/main/java/org/apache/seatunnel/connectors/seatunnel/iceberg/sink/writer/IcebergRecordWriter.java

```diff
@@ -91,6 +91,8 @@ record = recordConverter.convert(seaTunnelRow, rowType);
     public void applySchemaChange(SeaTunnelRowType afterRowType, SchemaChangeEvent event) {
         log.info("Apply schema change start.");
         SchemaChangeWrapper updates = new SchemaChangeWrapper();
+        // get the latest schema in case another process updated it
```

Comment:
cc @dailai . So iceberg support schema evolution too? I think we should update the doc.